examples/analog_out/cont_gen_voltage_wfm_int_clk_non_regen.py

```diff
@@ -0,0 +1,72 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to continuously generate an
+analog output waveform by providing new data to the output buffer
+as the task is running.
+
+This example is useful if you want to generate a non-repeating waveform,
+make updates on-the-fly, or generate a frequency that is not an
+even divide-down of your sample clock. In this example,
+the default frequency value is 17.0 to demonstrate that non-regenerative output
+can be used to create a signal with a frequency that is not an even divide-down
+of your sample clock.
+"""
+
+import numpy as np
+import numpy.typing
+
+import nidaqmx
+from nidaqmx.constants import AcquisitionType, RegenerationMode
+
+
+def create_sine_wave(
+    frequency: float, amplitude: float, sampling_rate: float, phase_in: float, phase_out: float
```

Comment:
phase_out should be a return value (using a tuple), not an input parameter. Callers should not have to calculate the phase.